app/assets/stylesheets/components/about.scss

```diff
@@ -38,6 +38,69 @@
     }
   }
 
+  #overview {
```

Comment:
Can we put this in its own style sheet please? that way it won't be nested and scoped to the rest of the about page styles 😄